### ISSUE TITLE
Top-down views of maze and normalization parameters

### ISSUE THREAD

**Issue by jaks19** [deepmind/gqn-datasets#10]
Hi I had 2 questions pertaining to the dataset:

1. The paper mentions 'top-down views' of the maze configurations. Are these views also included in the maze dataset, and if so, at which file indices would I be able to find them?

2. I am trying to normalize the datasets and finding their means and variances is taking about a full day per dataset. If the authors already have this information, would it be possible to know the mean and standard deviation of each training set, for each RGB channel?

Thanks for your help!
